services/graph/pkg/config/config.go

```diff
@@ -62,6 +62,8 @@ type LDAP struct {
 	UserNameAttribute        string `yaml:"user_name_attribute" env:"LDAP_USER_SCHEMA_USERNAME;GRAPH_LDAP_USER_NAME_ATTRIBUTE" desc:"LDAP Attribute to use for username of users."`
 	UserIDAttribute          string `yaml:"user_id_attribute" env:"LDAP_USER_SCHEMA_ID;GRAPH_LDAP_USER_UID_ATTRIBUTE" desc:"LDAP Attribute to use as the unique ID for users. This should be a stable globally unique ID like a UUID."`
 	UserEnabledAttribute     string `yaml:"user_enabled_attribute" env:"LDAP_USER_ENABLED_ATTRIBUTE;GRAPH_USER_ENABLED_ATTRIBUTE" desc:"LDAP Attribute to use as a flag telling if the user is enabled or disabled."`
+	LocalUserDisable         bool   `yaml:"local_user_disable" env:"LDAP_LOCAL_USER_DISABLE;GRAPH_LOCAL_USER_DISABLE" desc:"Boolean to control behavior for disabling users. If true, disabling a user via API will add the user to a group for disabled users."`
+	LocalUserDisableGroupDN  string `yaml:"local_user_disable_group_dn" env:"LDAP_LOCAL_USER_DISABLE_GROUP_DN;GRAPH_LOCAL_USER_DISABLE_GROUP_DN" desc:"Distinguished name of group to which added users will be classified as disabled when local_user_disable is set to true."`
```

Comment:
```suggestion
	LocalUserDisableGroupDN  string `yaml:"local_user_disable_group_dn" env:"LDAP_LOCAL_USER_DISABLE_GROUP_DN;GRAPH_LOCAL_USER_DISABLE_GROUP_DN" desc:"The Distinguished name of the group to which added users will be classified as disabled when the 'local_user_disable' setting is set to 'true'."`
```

Reply:
I'd suggest to name this `LDAP_DISABLED_USERS_GROUP_DN`.